RoslynPluginGenerator/DiagnosticAssemblyScanner.cs

```diff
@@ -71,7 +71,7 @@ private IEnumerable<DiagnosticAnalyzer> InstantiateDiagnosticsFromAssembly(strin
                     }
                     else
                     {
-                        this.logger.LogError(UIResources.Scanner_NoAnalyzers);
+                        this.logger.LogWarning(UIResources.Scanner_NoAnalyzers, analyzerAssembly.ToString());
```

Comment:
Miscellaneous change: previously, errors were being produced for any supporting assemblies in the package i.e. helper assemblies that didn't contain analyzers. Emitting a warning in this case is less confusing.